### PR TITLE
mod: Component and SceneModule .add .remove

### PR DESCRIPTION
**Pull request by BrookShuihuaLee**

### CODE REVIEW

**Comment by BrookShuihuaLee** [comment]
https://github.com/WhitestormJS/whs.js/issues/376

File: src/modules/app/SceneModule.js [hunk 1]
```diff
@@ -18,40 +20,51 @@ export class SceneModule {
...
+          if (object.parent !== this) return;
+      
+          await object.wait();
+      
+          object.parent = null;
+          self.scene.remove(object.native);
+          this.children.splice(this.children.indexOf(object), 1);
+        },
+        _setScene(scene) {
+          this.children = scene[SYMBOL_CHILDREN_FOR_SCENE] = scene[SYMBOL_CHILDREN_FOR_SCENE] || []
```
Comment (sasha240100) [inline src/modules/app/SceneModule.js:56]: Hey, thanks for the PR! Everything seems good. Can you explain me why do we use `Symbol` here? And I'll merge it 😀

Comment (BrookShuihuaLee) [reply src/modules/app/SceneModule.js:56]: The `scene` is a THREE.Scene object. We need store this.children in the object. If we use `scene._varName = this.children`, there is a risk that the three.js team may use `_varName` in THREE.Scene later. But in general, `Symbol` is safe.

Comment (sasha240100) [reply src/modules/app/SceneModule.js:56]: @BrookShuihuaLee Ok, thanks for explanation! Also, could you contact me in discord for further cooperation?

Comment (BrookShuihuaLee) [reply src/modules/app/SceneModule.js:56]: @sasha240100 ok, i've sent you a friend request 😄

**Pull request merged**